tests/accessor_legacy/accessor_api_image_common.h

```diff
@@ -1831,9 +1640,10 @@ class check_image_accessor_api_type {
     /**
      *  check image accessor api for all dimensions
      */
-    check_image_accessor_api_dim<T, 1>(log, typeName, count, size, queue);
-    check_image_accessor_api_dim<T, 2>(log, typeName, count, size, queue);
-    check_image_accessor_api_dim<T, 3>(log, typeName, count, size, queue);
+    // check_image_accessor_api_dim<T, 1>(log, typeName, count, size, queue);
```

Comment:
Why is this code commented out? It should be either fixed or removed.

Reply:
I'm not sure. I uncommented it, so let's see what CI has to say about it.
@andylshort, I'd like to hear your thoughts on this as this is your patch.

Reply:
As per the SYCL 2020 changes in [Appendix D.1](https://registry.khronos.org/SYCL/specs/sycl-2020/html/sycl-2020.html#sec:what-changed-between), "Support for the accessor targets `target::image` and `target::host_image` have been removed". This was causing these tests to fail, as shown in the above CI failure.

These `image` and `host_image` accessors aren't SYCL 2020-compliant and were causing various failures, hence why, even though these are categorised as "legacy" tests, I commented these out. Admittedly I could've provided better commit messages and done better, for which I apologise. This change fixed the bug and a discussion with my team thought this was sufficient enough due to these being legacy tests.

If a new PR is required to take this change further, or if changes need to be made here, do let me know.

Reply:
> I commented these out

I think it's a common practice in such cases to remove the code instead of commenting it out. Commented out code does more harm than good.